doc/classes/String.xml

```diff
@@ -181,6 +181,14 @@
 			<argument index="1" name="placeholder" type="String" default="&quot;{_}&quot;" />
 			<description>
 				Formats the string by replacing all occurrences of [code]placeholder[/code] with [code]values[/code].
+				[b]Note:[/b] There must be one and only one [code]_[/code] in the [code]placeholder[/code].
```

Comment:
Multiple underscores are allowed. Each will be replaced by the element key. Although I don't think there's any valid use case for it. The following snippet works:

```gdscript
print("{0000} {1111}".format(["Hello", "World"], "{____}"))  # Hello World
print("{ididid}".format({id=42}, "{___}"))  # 42
```

Reply:
It does work. I've written as the example given in the code comments before.